### ISSUE TITLE
Lazily loading Luxon library

### ISSUE THREAD

**Issue by musaffa** [cibernox/ember-power-calendar-luxon#3]
Luxon weighs around [~18KB](https://bundlephobia.com/result?p=luxon@1.12.0) (minified and gzipped). We can use lazy loading feature of `ember-auto-import` to decrease the vendor bundle size. Any thoughts?

**Comment by cibernox** [comment]
I don't think we can easily. If the page loads and there is a calendar on the page, it will try to render and luxon will have to be loaded. What would happen then? We don't have a way of disabling the calendar or show some kind of loading state while the luxon library hasn't loaded yet.

We'd have to create some changes to EPC first to allow lazy loading luxon/moment.

**Comment by musaffa** [comment]
Thanks. It makes sense. Anyways, looking forward to this feature in the future.
